.changeset/many-rice-begin.md

```diff
@@ -0,0 +1,21 @@
+---
+'@primer/react-brand': patch
+---
+
+⚠️ Breaking changes
+
+The `PricingOptions.FeatureListHeading` component has been renamed to `PricingOptions.FeatureListGroupHeading`. `PricingOptions.FeatureListHeading` is now used to optionally customize the heading of the `PricingOptions` feature list accordion.
+
```

Comment:
Can we add a diff here for those that are scanning this and need a quick before/after?

```diff
- <PricingOptions.FeatureListHeading />
+ <PricingOptions.FeatureListGroupHeading />
```